src/checkbox.scss

```diff
@@ -121,7 +121,8 @@ $block: #{$fd-namespace}-checkbox;
 
   &:indeterminate + .#{$block}__label--compact::before {
     content: "\e17b";
-    font-size: 0.5625rem;
+    font-size: 0.5rem;
+    padding: 0;
```

Comment:
Could you also fix the font-size of the icon in cozy (mobile) state? This is from the specs:
<img width="379" alt="Screen Shot 2021-04-28 at 12 25 40 PM" src="https://user-images.githubusercontent.com/39598672/116439395-7e373f80-a81d-11eb-81d4-7fdb202ab29d.png">

Currently we have 0.8125rem and a padding. I think that if you change it to 0.75rem you. won't need a padding and inthiscase you won't need to add like 125 for compact mode.
<img width="229" alt="Screen Shot 2021-04-28 at 12 30 54 PM" src="https://user-images.githubusercontent.com/39598672/116439501-9f982b80-a81d-11eb-94f8-84ef066004d1.png">

Reply:
@InnaAtanasova Done. Please re-check.